### PR TITLE
fix crash

### PR DESCRIPTION
**Pull request by frostbite169**
Issue #1 not elegant but I think it works. @JordanBundy  please check work.

### CODE REVIEW

**Comment by claraj** [comment]
"fix crash" - be more specific - which crash? "Prevent crash when adding duplicate book" would be better. 

Let's think about this one some more - is this the best solution?  The add book feature needs to report that either 
- the book was added
- the book was already in the database
If  the book is already in the database, how will the calling code know? 
Another way to think about this is that main can use try-except to catch that BookError. If no BookError, book was added. If BookError, book already in DB and **main** can display a message to the user.

See notes in the original issue https://github.com/claraj/readinglist/issues/21

Avoid printing from bookstore. bookstore's job is to manage the database, not talk to the user. Which module does handle talking to the user?

@JordanBundy what do you think?

**Comment by frostbite169** [comment]
I fixed what did what clara suggested

**Review by JordanBundy — COMMENTED** [review]
as far as i understand it, it looks good to me. i think it should be good to go.

File: main.py [hunk 1]
```diff
@@ -32,9 +32,11 @@ def create_menu():
...
 
 def add_book():
-    new_book = ui.get_book_info()
-    new_book.save()
-    
+    try:
+        new_book = ui.get_book_info()
+        new_book.save()
+    except BookError as e:
+        ui.message('book already added')
```
Comment (claraj) [inline main.py:39]: Grammar - '**B**ook already added.'

**Pull request merged**